docs/guides/python_chip_controller_building.md

```diff
@@ -255,429 +258,195 @@ the following stages:
-
-Start a scan action to search for valid CHIP devices over Bluetooth LE (for at
-most _timeout_ seconds). Stop when the device is matching the identifier or the
-counter times out.
-
-```
-chip-device-ctrl > ble-scan
-2021-05-29 22:28:05,461 ChipBLEMgr   INFO     scanning started
-2021-05-29 22:28:07,206 ChipBLEMgr   INFO     Name            = ChipLight
-2021-05-29 22:28:07,206 ChipBLEMgr   INFO     ID              = f016e23d-0d00-35d5-93e7-588acdbc7e54
-2021-05-29 22:28:07,207 ChipBLEMgr   INFO     RSSI            = -79
-2021-05-29 22:28:07,207 ChipBLEMgr   INFO     Address         = E0:4D:84:3C:BB:C3
-2021-05-29 22:28:07,209 ChipBLEMgr   INFO     Pairing State   = 0
-2021-05-29 22:28:07,209 ChipBLEMgr   INFO     Discriminator   = 3840
-2021-05-29 22:28:07,209 ChipBLEMgr   INFO     Vendor Id       = 9050
-2021-05-29 22:28:07,209 ChipBLEMgr   INFO     Product Id      = 20044
-2021-05-29 22:28:07,210 ChipBLEMgr   INFO     Adv UUID        = 0000fff6-0000-1000-8000-00805f9b34fb
-2021-05-29 22:28:07,210 ChipBLEMgr   INFO     Adv Data        = 00000f5a234c4e
-2021-05-29 22:28:07,210 ChipBLEMgr   INFO
-2021-05-29 22:28:16,246 ChipBLEMgr   INFO     scanning stopped
-```
-
-### `set-pairing-thread-credential <threadOperationalDataset>`
+### `SetThreadOperationalDataset(<thread-dataset>)`
 
 Provides the controller with Thread network credentials that will be used in the
 device commissioning procedure to configure the device with a Thread interface.
 
 ```
-chip-device-ctrl > set-pairing-thread-credential 0e080000000000010000000300001335060004001fffe002084fe76e9a8b5edaf50708fde46f999f0698e20510d47f5027a414ffeebaefa92285cc84fa030f4f70656e5468726561642d653439630102e49c0410b92f8c7fbb4f9f3e08492ee3915fbd2f0c0402a0fff8
+thread_dataset = bytes.fromhex("0e080000000000010000000300001335060004001fffe002084fe76e9a8b5edaf50708fde46f999f0698e20510d47f5027a414ffeebaefa92285cc84fa030f4f70656e5468726561642d653439630102e49c0410b92f8c7fbb4f9f3e08492ee3915fbd2f0c0402a0fff8")
+devCtrl.SetThreadOperationalDataset(thread_dataset)
 ```
 
-**REPL Commands:**
-`devCtrl.SetThreadOperationalDataset(bytes.FromHex("0e080000000000010000000300001335060004001fffe002084fe76e9a8b5edaf50708fde46f999f0698e20510d47f5027a414ffeebaefa92285cc84fa030f4f70656e5468726561642d653439630102e49c0410b92f8c7fbb4f9f3e08492ee3915fbd2f0c0402a0fff8"))`
-
-### `set-pairing-wifi-credential <ssid> <credentials>`
+### `SetWiFiCredentials(<ssid>: str, <password>: str)`
 
 Provides the controller with Wi-Fi network credentials that will be used in the
 device commissioning procedure to configure the device with a Wi-Fi interface.
 
 ```
-chip-device-ctrl > set-pairing-wifi-credential TESTSSID P455W4RD
+devCtrl.SetWiFiCredentials('TESTSSID', 'P455W4RD')
 ```
 
-**REPL Commands:** `devCtrl.SetWiFiCredentials('TESTSSID', 'P455W4RD')`
-
-### `connect -ip <address> <SetUpPinCode> [<nodeid>]`
+### `CommissionIP(<ip address>: str, <setup pin code>: int, <nodeid>: int)`
```

Comment:
This function is marked as deprecated in the API.

Reply:
+1, this is replaced with CommissionOnNetwork.

Reply:
Ok, since it is currently marked deprecated, we should drop it from here. But I do disagree on the deprecation of this API (see discussion in #tsg-tt-python-repl). But that is a different discussion.